blueprints/addon/files/.github/workflows/ci.yml

```diff
@@ -33,6 +33,7 @@ jobs:
   floating:
     name: "Floating Dependencies"
     runs-on: ubuntu-latest
+    timeout-minutes: 5
```

Comment:
let's make this 10min, that seems like a better default than 5 to me

Reply:
So I was feeling that this was probably a good idea for the **app** tests but I can see a benefit for the default to be the same for everything. This PR really isn't designed to cut things off in the most efficient way, just to prevent the absolute worst case scenario, and 10 mins is a lot less than 6 hours 😂